### PR TITLE
Feat 421 audit log generic

### PR DESCRIPTION
**Pull request by KZavorotna**
Related PRs:
https://github.com/withthegrid/platform-client/pull/494
https://github.com/withthegrid/platform/pull/669

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
     "@types/qs": "^6.9.6",
     "axios": "^0.21.1",
     "joi": "^17.4.0",
-    "qs": "^6.10.1"
+    "qs": "^6.10.1",
+    "rimraf": "^3.0.2"
```
Comment (everhardt) [inline package.json:33]: Can you put rimraf in the devDependencies? No need to have it in the implementing repo's (platform-client for example) I think?

File: src/models/log.ts [hunk 2]
```diff
@@ -3,8 +3,8 @@ import Joi from 'joi';
 const schema = Joi.object().keys({
   hashId: Joi.string().required().example('op09a'),
   userHashId: Joi.string().allow(null).required().example('b45zo0'),
-  objectType: Joi.string().allow(null).required().example('command'),
-  objectHashId: Joi.string().allow(null).required().example('ga9741s'),
+  objectType: Joi.string().allow(null).example('command'),
+  objectHashId: Joi.string().allow(null).example('ga9741s'),
```
Comment (everhardt) [inline src/models/log.ts:7]: This model is only used for responses from the server. I think those will always contain these two properties, so they will still have to be required

**Review by everhardt — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (KZavorotna) [reply package.json]: Updated

File: src/models/log.ts (hunk 2 above)
Comment (KZavorotna) [reply src/models/log.ts]: Updated

**Comment by KZavorotna** [comment]
PR is updated

**Pull request merged**